.github/workflows/manual-docker-prune.yml

```
@@ -0,0 +1,10 @@
+name: Manual Docker System Prune (CDCgov)
+on:
+  workflow_dispatch:
+
+jobs:
+  docker-system-prune:
+    runs-on: cfa-cdcgov
+
+    steps:
+    - run: docker system prune -a -f
```

Micah did you also add volumes?
```suggestion
    - run: docker system prune --all --force --volumes
```